src/include/storage/bufmgr.h

```diff
@@ -86,8 +86,8 @@ extern PGDLLIMPORT int NLocBuffer;
 extern PGDLLIMPORT Block *LocalBufferBlockPointers;
 extern PGDLLIMPORT int32 *LocalRefCount;
 
-/* upper limit for effective_io_concurrency */
-#define MAX_IO_CONCURRENCY 1000
+/* upper limit for effective_io_concurrency (better to he power of 2) */
+#define MAX_IO_CONCURRENCY 1024
```

Comment:
I do no think that this change significantly affect compatibility with Vanilla.
Why it is needed? Because I am using it for division by modulo in to access ring buffer.
Even of modern CPUs DIV instruction is order of magnitude slower than bit shift.
Is is so critical? Do not think so, especially taken in account that compiler replaces division by constant with multiplication.
But in any case...  Postgres is not using this values other than as upper boundary for correspondent GUC.
So why not to change it to power of two which may save some CPU ticks?